tests/components/myuplink/test_switch.py

```diff
@@ -0,0 +1,80 @@
+"""Tests for myuplink switch module."""
+
+from unittest.mock import MagicMock
+
+from homeassistant.components.switch import DOMAIN as SWITCH_DOMAIN
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    SERVICE_TURN_OFF,
+    SERVICE_TURN_ON,
+    STATE_OFF,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_platform
+
+from tests.common import MockConfigEntry
+
+ENTITY_ID = "switch.f730_cu_3x400v_temporary_lux"
+ENTITY_FRIENDLY_NAME = "F730 CU 3x400V Tempo­rary lux"
+ENTITY_UID = "batman-r-1234-20240201-123456-aa-bb-cc-dd-ee-ff-50004"
+
+
+async def test_entity_registry(
+    hass: HomeAssistant,
+    entity_registry: er.EntityRegistry,
+    mock_myuplink_client: MagicMock,
+    mock_config_entry: MockConfigEntry,
+) -> None:
+    """Tests that the devices are registered in the entity registry."""
+    await setup_platform(hass, mock_config_entry, SWITCH_DOMAIN)
```

Comment:
We can set the platform fixture like this or in a fixture defined in this module that is auto used by all tests in this module.

https://github.com/home-assistant/core/blob/0d6f4058ca61b97793f816109a402889a03e162a/tests/components/evil_genius_labs/test_light.py#L16